src/features/singleRight/components/ResourceActionBar/ResourceActionBar.module.css

```
@@ -29,7 +29,3 @@
   padding-right: 12px;
   color: var(--fds-semantic-text-neutral-subtle) !important;
 }
```

I think something must have gone wrong with the git pulling somehow.
Please add back the changes to main that were merged last week
```suggestion
}

.actionBarButtonText {
  white-space: nowrap;
}
```